plugins/source/github/resources/services/repositories/sboms.go

```diff
@@ -0,0 +1,35 @@
+package repositories
+
+import (
+	"context"
+
+	"github.com/cloudquery/cloudquery/plugins/source/github/client"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/google/go-github/v59/github"
+)
+
+func sboms() *schema.Table {
+	return &schema.Table{
+		Name:      "github_repository_sboms",
+		Resolver:  fetchSboms,
+		Multiplex: client.OrgRepositoryMultiplex,
+		Transform: client.TransformWithStruct(&github.SBOMInfo{}),
```

Comment:
```suggestion
		Transform: client.TransformWithStruct(&github.SBOMInfo{}, transformers.WithPrimaryKeys("SPDXID")),
```

Per my previous comment we should do this or add the version too